hazelcast/src/main/java/com/hazelcast/collection/impl/queue/operations/QueueOperation.java

```diff
@@ -63,15 +63,14 @@ protected QueueOperation(String name, long timeoutMillis) {
     }
 
     protected final QueueContainer getOrCreateContainer() {
-        if (container == null) {
-            QueueService queueService = getService();
-            try {
-                container = queueService.getOrCreateContainer(name, this instanceof BackupOperation);
-            } catch (Exception e) {
-                throw new RetryableHazelcastException(e);
-            }
+        QueueContainer queueContainer;
+        QueueService queueService = getService();
+        try {
+            queueContainer = queueService.getOrCreateContainer(name, this instanceof BackupOperation);
+        } catch (Exception e) {
```

Comment:
The original code already does it; but do we want to retry on any exception? Imagine we have a NPE or a ClassNotFoundException. There is no point in retrying.